.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Special notes for your reviewer**:
+
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
```

Comment:
@gyliu513 I don't understand what this means.

Reply:
Killed this, this is get from other projects, which depend on some images in the repo, but for this, I think we can remove it.